grafana/dashboards/harvest_dashboard_node_details.json

```diff
@@ -2355,7 +2347,7 @@
             "alertThreshold": true
           },
           "percentage": false,
-          "pluginVersion": "7.5.6",
+          "pluginVersion": "8.1.1",
```

Comment:
not changed in this review but FabricPool doesn't have any sub charts in this dashboard